### ISSUE TITLE
Sound browser sample numbering does not match superdirt

### ISSUE THREAD

**Issue by cleary** [tidalcycles/atom-tidalcycles#122]
In the new sound browser, if a sample has special characters in it's filename the reference index of the sample in the browser does not match the sample played by superdirt.

Simple example on `pluck:1`

**Comment by ndr-brt** [comment]
Not only, I noticed that files with special characters (like `pluck/BS A#2 PI.wav` doesn't play at all)

**Comment by ndr-brt** [comment]
I checked...
Seems like javascript order follows the "ASCII" sort, so, the files
```
'BS A#2 PI.wav'
'BS A1 PI.wav'
```
are already sorted, because the `#` is smaller than any number.

But, don't know in other OS, but in linux the same two files are ordered, at OS level as:
```
'BS A1 PI.wav'
'BS A#2 PI.wav'
```

SuperDirt doesn't apply a particular order, but it inherits it from the OS.

The possible solutions are:
- force plugin to use OS sorting
- force superdirt to use ASCII sorting

The first will be the best for me, but we should verify how to apply that sorting rules.

**Comment by yaxu** [comment]
Yes it woudl be good to resolve this. There's an issue open here: https://github.com/musikinformatik/SuperDirt/issues/131
